addons/mail/static/src/model/model_field.js

```diff
@@ -622,9 +678,11 @@ class ModelField {
      *
      * @private
      * @param {mail.model} record
+     * @param {Object} param1
+     * @param {boolean} param1.hasToUpdateInverse
      * @returns {boolean} whether the value changed for the current field
      */
-    _setRelationUnlinkX2One(record) {
+    _setRelationUnlinkX2One(record, { hasToUpdateInverse }) {
         const toDeleteOtherRecord = this.read(record);
```

Comment:
would rename "delete" to "unlink"

Reply:
I think the original meaning what to add/delete "into the relation set", which was correct. But ok to use link/unlink, at least it's correct no matter how we look at it.